agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -84,9 +85,9 @@ const (
 
 	// retry settings for pulling images
 	maximumPullRetries        = 5
-	minimumPullRetryDelay     = 250 * time.Millisecond
-	maximumPullRetryDelay     = 1 * time.Second
-	pullRetryDelayMultiplier  = 1.5
+	minimumPullRetryDelay     = 1100 * time.Millisecond
+	maximumPullRetryDelay     = 5 * time.Second
+	pullRetryDelayMultiplier  = 2
```

Comment:
Are these to make the tests which depend on pulling images from ECR less flaky?

Reply:
Right, the longer interval will make sure that if user gets throttled by ECR. the next retry will jump over a throttle bucket (1s). With the current setup starting with 0.25s, the user will be throttled again because the next retry is still within the same throttle bucket.